### ISSUE TITLE
Support for Pass-Through queries

### ISSUE THREAD

**Issue by rakusan2** [timabell/msaccess-vcs-integration#91]
I have an application that has a Pass-Through Query 
(It is a complex query that takes a whole minute to run local but less than tenth of a second on the PostgreSQL server)
As of now if I do import I get
```
Importing queries...    VCS_ImportExport.ImportProject: Error #3075
Syntax error in query expression 'COALESCE(
```
Which most likely happens do it it being imported as if it was local

**Comment by hecon5** [comment]
I've made a few tweaks to the VCS_Queries.bas file; it supports export/import for "SQL" queries and the pass through capabilities. Basically, it sets the ".connect" string, which seems to tell Access to not try and error check the query. I'm just cleaning it up, and I'll comment here again with the pull request info.

**Comment by hecon5** [comment]
https://github.com/timabell/msaccess-vcs-integration/pull/92 is the relevant PR.

**Comment by hecon5** [comment]
Note these changes are also based off Hangie's fork, too, but I've not seen any issues yet with the VCS_Queries IO functions.

**Comment by timabell** [comment]
Resolved in #92
